controllers/gclient/gclient.go

```diff
@@ -104,12 +105,17 @@ func (g *GClient) CreateTicket(t GithubTicket) error {
 		return err
 	}
 
-	res, err := sendRequest("POST", t.RepositoryURL+"/issues", requestBody)
+	requestUrl, err := g.getAPIBaseURL(t.RepositoryURL)
+	if err != nil {
+		return err
+	}
+	requestUrl += "/issues"
```

Comment:
Possibly worth creating a `getURLforAPI(t.RepositoryURL, "/issues")` function

Reply:
I agree, the composition of "some URL" is repeated in all the other APIs, and this will be reusable